src/common/gui/SurgeGUIEditor.cpp

```diff
@@ -2636,6 +2641,72 @@ void SurgeGUIEditor::showSettingsMenu(CRect &menuRect)
     eid++;
     mpeSubMenu->forget();
 
+    // Mouse behavior
+
+    int mid = 0;
+
+    COptionMenu* mouseSubMenu =
+        new COptionMenu(menuRect, 0, 0, 0, 0, VSTGUI::COptionMenu::kNoDrawStyle);
```

Comment:
As you'll see in a second probably want to make this `NoDrawStyle | MultiCheckStyle`